src/openrct2/rct1.h

```diff
@@ -30,6 +30,12 @@
 #define RCT1_MAX_MAP_ELEMENTS       0xC000
 #define RCT1_MAX_SPRITES            5000
 #define RCT1_MAX_VEHICLES_PER_RIDE  12
+#define RCT1_MAX_STATIONS           4
```

Comment:
Please move this to rct12.h and rename it to RCT12_MAX_STATIONS_PER_RIDE.

The same applies to all constants that are the same in both RCT1 and RCT2.